release-notes/latest.md

```diff
@@ -1,3 +1,25 @@
+#### <sub><sup><a name="4708" href="#4708">:link:</a></sup></sub> feature
+
+* The first step (heh) along our [road to v10](https://blog.concourse-ci.org/core-roadmap-towards-v10/) has been taken!
+
+  @evanchaoli implemented the [`set_pipeline` step](https://concourse-ci.org/set-pipeline-step.html) described by [RFC #31](https://github.com/concourse/rfcs/pull/31). The RFC is still technically in progress so the step is 'experimental' for now.
+
+  The `set_pipeline` step allows a build to configure a pipeline within the build's team. This is the first "core" step type added since the concept of "build plans" was introduced, joining `get`, `put`, and `task`. Exciting!
+
+  The key goal of the v10 roadmap is to support multi-branch and PR workflows, which require something more "dynic" than `fly set-pipeline`. The theory is that by making pipelines more first-class - allowing them to be configured and automated by Concourse itself - we can support these more dynamic use cases by leveraging existing concepts instead of adding complexity to existing ones.
```

Comment:
```suggestion
  The key goal of the v10 roadmap is to support multi-branch and PR workflows, which require something more "dynamic" than `fly set-pipeline`. The theory is that by making pipelines more first-class - allowing them to be configured and automated by Concourse itself - we can support these more dynamic use cases by leveraging existing concepts instead of adding complexity to existing ones.
```